### ISSUE TITLE
Improvements to tutorials

### ISSUE THREAD

**Issue by mirams** [Chaste/Chaste#224]
Now the tutorials are on the new website, following #193 , time for a spring clean. 

A few improvements we noted during the migration:
## Top Priority
- [x] Specify where test output goes, both in general & in docker (@bdevans). Could either be on the first tutorial https://chaste.github.io/docs/user-tutorials/writingtests/  or mentioned in big letters on the tutorial index page (I probably prefer the latter).

## Nice to have
- [ ] Move java visualiser code to separate repo (code lives in the anim folder). Keep `anim` folder but replace code with a readme pointing to its new location.
- [ ] Remove references to java visualiser from tutorials / docs.
- [ ] Update paraview tutorial to make it a lot more novice-user friendly.

**Comment by bdevans** [comment]
So we recently renamed the defunct `test_summary` folder in the main repo to `output` so that there is a known, useable folder in the repo that VSCode can automatically use as a mount when someone clones the repo and opens it in a devcontainer. For uniformity, I also changed the environment variable `CHASTE_TEST_OUTPUT="${CHASTE_DIR}/output"` in the docker images to match (i.e. by default it will be `/home/chaste/output`). This way the path is shorter and more general too as people could produce other types of output there (and could if they wish, still have a subfolder for test outputs etc.).

**Comment by mirams** [comment]
OK, good, in that case it might be good enough to make the tutorials refer to `CHASTE_TEST_OUTPUT` and not need any docker specialisation.

**Comment by bdevans** [comment]
I agree - a level of abstraction by using an environment variable makes it all a lot clearer, more uniform and much more flexible.
